service/static/js/rest_api.js

```diff
@@ -404,6 +454,7 @@ $(function () {
             table += '<th class="col-md-2">Last Name</th>'
             table += '<th class="col-md-2">Email</th>'
             table += '<th class="col-md-2">Password</th>'
+            table += '<th class="col-md-2">Add_Id</th>'
```

Comment:
Add_ID just to be consistent with Cust_ID.

Reply:
Fixed